draft-ietf-quic-recovery.md

```diff
@@ -1434,26 +1411,27 @@ increases bytes_in_flight.
 ## On Packet Acknowledgement
 
 Invoked from loss detection's OnPacketAcked and is supplied with the
-acked_packet from sent_packets.
+newly acked_packets from sent_packets.
 
 ~~~
-   InCongestionRecovery(sent_time):
+   InLossRecovery(sent_time):
```

Comment:
```suggestion
   InCongestionRecovery(sent_time):
```
Revert this change. Was this deliberate? The term congestion recovery is used earlier in the spec and the term is defined.

Reply:
This was deliberate.  I've always heard it called loss recovery or just 'the recovery period'

But I can revert it.